File: find_history.js
const fs = require('fs');
const path = require('path');

const historyDir = 'C:\\Users\\Administrator\\AppData\\Roaming\\Code\\User\\History';
let allFiles = [];

function walkDir(dir) {
    if (!fs.existsSync(dir)) return;
    const files = fs.readdirSync(dir);
    for (const f of files) {
        const fullPath = path.join(dir, f);
        const stat = fs.statSync(fullPath);
        if (stat.isDirectory()) {
            walkDir(fullPath);
        } else if (f !== 'entries.json') {
            allFiles.push({
                path: fullPath,
                mtime: stat.mtimeMs,
                mtimeStr: stat.mtime.toLocaleString(),
                size: stat.size
            });
        }
    }
}

walkDir(historyDir);
allFiles.sort((a, b) => b.mtime - a.mtime);

for (let i = 0; i < Math.min(30, allFiles.length); i++) {
    const f = allFiles[i];
    console.log(`${f.path} - ${f.mtimeStr} - ${f.size} bytes`);
}

File: find_history.py
import os
import time

history_dir = r'C:\Users\Administrator\AppData\Roaming\Code\User\History'
all_files = []

for root, dirs, files in os.walk(history_dir):
    for f in files:
        if f != 'entries.json':
            full_path = os.path.join(root, f)
            all_files.append(full_path)

all_files.sort(key=os.path.getmtime, reverse=True)

for f in all_files[:30]:
    size = os.path.getsize(f)
    mtime = time.ctime(os.path.getmtime(f))
    print(f"{f} - {mtime} - {size} bytes")
